New package: QED v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: QED
- Repository: https://github.com/QEDjl-project/QED.jl
- Created by: @AntonReinhard
- Version: v0.1.0
- Commit: c43e326e7c2fbe4829b22560dc06320500cdeb3a
- Reviewed by: @AntonReinhard
- Reference: https://github.com/QEDjl-project/QED.jl/issues/50#issuecomment-2325958089
- Description: [WIP] QED.jl: A strong-field particle physics framework
<!-- bf0c69308befbd3ccf2cc956ac8a4671113b158fd87a6d4e741b0173dab051a48f7dc43e593ffff4357bc647a3a83598a26e7f2616c478198835dfe96c96e6c57dcd7011a747369c680b16a4d4eea2d9d76880e9e16b0d4c72e422ade0eb81aaf8447d0f50abd83dfacdc6e7f0e014b2565d6e67e588376a085fff40826ade22ad8398e93e0da698f369238c107068ef76b2b9ab4eee62361a5bba9f616ee295b0579c451a28346aa8167e7b5e4ebe67 -->

AntonReinhard commented:
[noblock] Oh, it seems the last documentation update wasn't properly pushed to gh-pages. I'll try to fix this. As for the name, we're aware this doesn't fit the normal requirements, but since the individual packages are all called `QEDsomething.jl` it would be nice to have the overall package be just `QED.jl`.

goerz commented:
> Since the individual packages are all called QEDsomething.jl it would be nice to have the overall package be just QED.jl

I'd consider that a pretty strong argument. With "QED" being pretty widely understood at least by physicists, names like `QEDprocesses` and `QEDfields.jl` are great package names (although I would have recommended the more standard CamelCase `QEDProcesses` and `QEDfields` – alas, they're already registered). Thus, I would support `QED` to tie these packages together as a framework. There's a pretty strong bias against acronyms for Julia packages, and *especially* three-letter-acronyms. However, I wouldn't consider `QED` to be an acronym in the same sense, given that it is a well-known physics concept, and a nice pun (I think, at least, I never actually looked up who coined the term).

With package names like `QED`, it also works in favor of the registration if the package is already reasonably polished and is well-maintained. Multiple contributores, and maintainers with a permanent position or backing by a tenured PI or a company go a long way here. From the contribution summary, that seems to be the case here.

In any case, you'll have to ask for a manual merge on Slack after the three-day waiting period.

[noblock]

JuliaRegistrator committed:
New package: QED v0.1.0

UUID: bb1fba1d-cf9b-41b3-874e-4b81465537b9
Repo: https://github.com/QEDjl-project/QED.jl.git
Tree: 2aca53f6a68ab2aa434899383d74a16b2c474b9e

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a

goerz commented:
Followup in https://github.com/JuliaRegistries/General/pull/115082
